<launch>
    <arg name="att_controller" default="pid" />
    <node
        name="control"
        pkg="control"
        type="control.py"
        output="screen">
        <param name="config_file" value="control_params.yaml" />
        <param name="attitude_controller" value="$(arg att_controller)"/>
    </node>
</launch>